lib/systems/inspect.nix

```
@@ -13,6 +13,17 @@ rec {
     isx86_64       = { cpu = { family = "x86"; bits = 64; }; };
     isPower        = { cpu = { family = "power"; }; };
     isPower64      = { cpu = { family = "power"; bits = 64; }; };
+    # This architecture is the default in NixOS PowerPC64 BE, but not on
+    # mainline GCC, so it sometimes causes issues in certain packages that
+    # makes the wrong assumption on the used ABI.
+    isPower64BeElfv2 = {
+      cpu = {
+        family = "power";
+        bits = 64;
+        significantByte = significantBytes.bigEndian;
+      };
+      abi = { abi = "elfv2";};
+    };
```

> It also adds a convenience `isPower64BeElfv2` function, since a few downstream packages needs patches for this specific case.

`inspect.nix` is pretty crowded already; can they use `with stdenv.hostPlatform; isPower64 && isBigEndian && isAbiElfv2`?

```suggestion
    isAbiElfv2 = { abi = { abi = "elfv2"; }; };
```